src/main/java/com/hubspot/jinjava/lib/tag/PrintTag.java

```diff
@@ -26,9 +26,9 @@ public String getName() {
   }
 
   @Override
-  public String interpret(TagNode tagNode,
-      JinjavaInterpreter interpreter) {
-    return Objects.toString(interpreter.resolveELExpression(tagNode.getHelpers(), tagNode.getLineNumber()), "");
+  public String interpret(TagNode tagNode, JinjavaInterpreter interpreter) {
+    String result = Objects.toString(interpreter.resolveELExpression(tagNode.getHelpers(), tagNode.getLineNumber()), "");
+    return interpreter.getContext().isValidationMode() ? result : "";
```

Comment:
Is the `PrintTag` for `ValidationMode` only?

Reply:
Nope, it's a little-known tag. 

`{% print "hello" %}`